packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -777,8 +778,10 @@ export abstract class SharedObject<
 		incrementBy: number,
 		telemetryContext?: ITelemetryContext,
 	) {
-		const prevTotal = (telemetryContext?.get(this.telemetryContextPrefix, propertyName) ??
-			0) as number;
+		const prevTotal = ((telemetryContext as TelemetryContext)?.get(
```

Comment:
We can be more type safe here. I think this should work, can you verify? Will need to import our `assert` function.
```suggestion
		// TelemetryContext is the only current (and planned) implementation of ITelemetryContext
		assert(telemetryContext instanceof TelemetryContext, "received context is not an instance of TelemetryContext");
		const prevTotal = (telemetryContext.get(
```

Reply:
Updated